data/.gitignore

```diff
@@ -3,4 +3,3 @@
 /prepared-data.csv
 /prepared-data-train.csv
 /prepared-data-test.csv
-/matrix-train.joblib
```

Comment:
or `*.joblib` or something?

Reply:
I don't understand what you are trying to say here.

Reply:
I mean to ignore everything with that extension (or `.pkl`)

Reply:
Ok now I see. The `.gitignore` files in the subfolder are automatically edited by dvc so we shouldn't usually touch them. As soon as we add a *.pkl to the dvc pipeline, it will be automatically be ignored.